### DISCUSSION TITLE
Presence.Read.All Permission is missing

### DISCUSSION THREAD

**Discussion by bckrma** [microsoft-search/pnp-modern-search#3365] [Q&A] [answered]
Hello,
i am working on an employee directory using the pnp modern search version 4.9.3.
I would like to display the teams presence status with a <mgt-person> card like this:

```
<mgt-person user-id="{{getUserEmail (slot item @root.slots.PersonQuery)}}" person-card="hover" show-presence>
</mgt-person>
```
As far as i understand I need to add the graph permission "Presence.Read.All" to the webpart to do this.
But i cant find it in the API access page in the sharepoint admin center.
I checked on of my test tenants in which version 4.8 is still installed, and the "Presence.Read.All" permission is listed in the API access page.

Do i need to add this graph permission manually? If yes, is there some kind of documentation on how to do this?

Thanks a lot!

**Comment by kasperbolarsen** [comment, marked as answer]
I have just created a brand new Demo tenant and installed PnP Modern Search v 4.9.3
And that permission is right there. Have you checked if it is in Pending requests/approved requests?

![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/6bb1e6d0-55b9-4e3b-b111-453c99cb1a56)

**Reply by bckrma** [reply]
Argh, my bad. Sorry i have overlooked the permission 😬 
But thanks for checking anyways!
